schema/deploy/tables/project.sql

```diff
@@ -1,12 +1,14 @@
 -- Deploy cif:tables/project to pg
 -- requires: tables/funding_stream
+-- requires: tables/funding_stream_rfp
 
 begin;
 
 create table cif.project(
   id integer primary key generated always as identity,
   operator_id integer references cif.operator(id) not null,
   funding_stream_id integer not null references cif.funding_stream(id),
+  funding_stream_rfp integer not null references cif.funding_stream_rfp(id),
```

Comment:
```suggestion
  funding_stream_rfp_id integer not null references cif.funding_stream_rfp(id),
```

Reply:
Do we need the `funding_stream_id` in this table if we have a foreign key to a `funding_stream_rfp`?

We might want to rethink the form component so that it allows you to the stream and rfp year, but only sets the  `fundingStreamRfpId` in the form.

Reply:
This still need a rename to `funding_stream_rfp_id` to be consistent with our naming conventions

Reply:
:point_up: